BuildResidentialScheduleFile/resources/constants.rb

```diff
@@ -29,6 +29,18 @@ def self.PlugLoadsOtherMonthlyMultipliers
     return '1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248'
   end
 
+  def self.PlugLoadsTVWeekdayFractions
+    return '0.014, 0.007, 0.004, 0.003, 0.004, 0.006, 0.010, 0.015, 0.020, 0.025, 0.028, 0.031, 0.033, 0.038, 0.042, 0.046, 0.054, 0.062, 0.080, 0.110, 0.132, 0.125, 0.077, 0.034'
+  end
+
+  def self.PlugLoadsTVWeekendFractions
+    return '0.014, 0.007, 0.004, 0.003, 0.004, 0.006, 0.010, 0.015, 0.020, 0.025, 0.028, 0.031, 0.033, 0.038, 0.042, 0.046, 0.054, 0.062, 0.080, 0.110, 0.132, 0.125, 0.077, 0.034'
+  end
```

Comment:
Should we adopt the separate "ATUS TVs weekday" and "ATUS TV weekend" schedule fractions from which these "average" schedule fractions were derived?

Reply:
Looks like most other categories have the same schedule fractions between weekday and weekend.

Reply:
Seems like a question for @rajeee or @afontani.

Reply:
@shorowit : I could have sworn we had weekday/weekend schedules broken out at one point. Maybe I am getting confused with the stochastic schedule generator. There are significant schedule differences between weekday and weekends for TVs (and other schedules). I would like the schedules broken out by weekday and weekend. To me, it seems relevant to better capture reality, but I understand if we stay consistent with ANSI/RESNET/ICC 301.